HalideGenerator.cmake

```diff
@@ -75,7 +75,7 @@ function(halide_add_generator_dependency)
   # Add a custom target to invoke the GENERATOR_TARGET and output the Halide
   # generated library
   if (WIN32)
-    add_custom_command(OUTPUT "${FILTER_LIB}" "${FILTER_HDR}"
+    add_custom_command(OUTPUT "${SCRATCH_DIR}/${FILTER_LIB}" "${SCRATCH_DIR}/${FILTER_HDR}"
```

Comment:
I just tried this change on top of my branch with Visual Studio 2013. It seemed to build fine.